build.gradle

```diff
@@ -858,17 +857,45 @@ task product(type: Zip) {
         exclude '.git*'
       }
     }
-    if (isEnterpriseProduct && hasAqpProject) {
-      // copy enterprise shared libraries for optimized JNI calls
-      copy {
-        from aqpProject.projectDir.path + '/lib'
-        into "${snappyProductDir}/jars"
+    if (isEnterpriseProduct) {
+      if (hasAqpProject) {
+        // copy enterprise shared libraries for optimized JNI calls
+        copy {
+          from aqpProject.projectDir.path + '/lib'
+          into "${snappyProductDir}/jars"
+        }
+        copy {
+          from aqpProject.projectDir
+          into snappyProductDir
+          include 'NOTICE'
+          include '*EULA*'
+        }
       }
-      copy {
-        from aqpProject.projectDir
-        into snappyProductDir
-        include 'NOTICE'
-        include '*EULA*'
+
+      def jdbcConnectorProject = project(":snappy-jdbc-connector_${scalaBinaryVersion}")
+      def gemfireConnectorProject = project(":gemfire-connector")
+      def gfeConnectorProject = project(":gemfire-connector:connector_${scalaBinaryVersion}")
+      def gfeFunctionProject = project(":gemfire-connector:gfeFunctions")
+      if (hasJdbcConnectorProject) {
+        copy {
+          from jdbcConnectorProject.jar.destinationDir
+          into "${snappyProductDir}/connectors"
+        }
+      }
+      if (hasGemFireConnectorProject) {
+        copy {
+          from gfeConnectorProject.jar.destinationDir
+          into "${snappyProductDir}/connectors"
+        }
+        copy {
+          from gfeFunctionProject.jar.destinationDir
+          into "${snappyProductDir}/connectors"
+        }
+        copy {
+          from "${gemfireConnectorProject.projectDir}/examples/quickstart/data"
+          into "${snappyProductDir}/connectors"
+          include "persons.jar"
```

Comment:
Shouldn't we place in the quickstart folder. May be main quickstart. Copying here in connector seems odd.

Reply:
As that's required by the connector project, thought it'll be better to have it in one place. @ahshahid ?

We also place JDBC client jar in here. Can restructure the directory into connectors/jdbc, connectors/gemfire, connectors/gemfire/quickstart and connectors/jdbc.